NIPTool/server/web_app/api/api_v1/endpoints/update.py

```diff
@@ -1,66 +1,79 @@
+import logging
+from datetime import datetime
+from typing import Iterable
+
 from fastapi import APIRouter, Depends, Request
 from fastapi.responses import RedirectResponse
-
 from NIPTool.adapter.plugin import NiptAdapter
-from NIPTool.server.web_app.utils import *
+from NIPTool.crud import find, update
+from NIPTool.models.database import Sample, User
 from NIPTool.server.web_app.api.deps import get_nipt_adapter
-from datetime import datetime
+from NIPTool.server.web_app.utils import *
 
 router = APIRouter()
 
+LOG = logging.getLogger(__name__)
+
 
```

Comment:
This whole function will be re written and split into smaller. But lets do it later

Reply:
Sound good